### ISSUE TITLE
EulerAncestralDiscreteScheduler produces undesired results with Stable Diffusion pipeline

### ISSUE THREAD

**Issue by zxydi1992** [huggingface/diffusers#2941]
### Describe the bug

The generated images are not natural and too blurry.

### Reproduction

scheduler = EulerAncestralDiscreteScheduler()
pipe = StableDiffusionPipeline.from_pretrained("CompVis/stable-diffusion-v1-4", scheduler=scheduler)
pipe = pipe.to('cuda')

# example 1
pipe('A dog eat on the pizza').images[0]

# example 2
wrapper('A blue cat plays with a red dog').images[0]

### System Info

- `diffusers` version: 0.15.0.dev0
- Platform: Linux-5.15.0-1015-aws-x86_64-with-glibc2.10
- Python version: 3.8.5
- PyTorch version (GPU?): 1.12.1 (True)
- Huggingface_hub version: 0.13.3
- Transformers version: 4.27.4
- Accelerate version: 0.18.0
- xFormers version: not installed
- Using GPU in script?: <fill in>
- Using distributed or parallel set-up in script?: <fill in>

**Comment by takuma104** [comment]
Hi @zxydi1992, How about changing the code like this? This should correctly configure the EulerAncestralDiscreteScheduler.
```py
pipe = StableDiffusionPipeline.from_pretrained("CompVis/stable-diffusion-v1-4")
pipe.scheduler = EulerAncestralDiscreteScheduler.from_config(pipe.scheduler.config)
pipe.to('cuda')
```

**Comment by sayakpaul** [comment]
@zxydi1992 yes https://github.com/huggingface/diffusers/issues/2941#issuecomment-1492481449 should cut it. Thanks, @takuma104!
